src/routes/(dashboard)/ManagerDashboard.svelte

```diff
@@ -166,29 +169,34 @@
         }
 
         if (
-            ($searchParams.tab === Tab.manage && !manageLastAssignedNames.includes($searchParams.lastAssigned)) ||
-            ($searchParams.tab === Tab.myWork && !myWorkLastAssignedNames.includes($searchParams.lastAssigned))
+            ($searchParams.tab === Tab.manage &&
+                !manageLastAssignedUsers.filter((u) => u.id === $searchParams.lastAssignedId)) ||
```

Comment:
I think we need to add a `.length` after the filter here since the filter's result will always be an array which will be truthy.